extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -382,6 +382,14 @@ export class AmpIframe extends AMP.BaseElement {
       return;
     }
 
+    if (height < 100) {
+      user.warn(TAG_,
+          'ignoring embed-size request because the resize height is ' +
+          'less than 100px',
+          this.element);
```

Comment:
Should both be .error

On Thu, Jul 7, 2016 at 3:51 PM, Yuxi Chen <notifications@github.com> wrote:

> In extensions/amp-iframe/0.1/amp-iframe.js
> <https://github.com/ampproject/amphtml/pull/3954#discussion_r70001715>:
>
> > @@ -382,6 +382,14 @@ export class AmpIframe extends AMP.BaseElement {
> >        return;
> >      }
> >
> > +    if (height < 100) {
> > +      user.warn(TAG_,
> > +          'ignoring embed-size request because the resize height is ' +
> > +          'less than 100px',
> > +          this.element);
>
> We use user.warn for resize request on a non-resizable amp-iframe. So I
> used the same log level. @cramforce <https://github.com/cramforce> What
> do you think?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ampproject/amphtml/pull/3954/files/06a7b752d455261f4d83f6b291104db56703bb20#r70001715>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAFeT_jpEenrdtRFEc4-jymPys9Dd0kxks5qTYMAgaJpZM4JHXki>
> .
>